.vscode/settings.json

```diff
@@ -5,10 +5,13 @@
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
     "python.testing.cwd": "${workspaceFolder}/backend",
-    "mypy.configFile": "./backend/mypy.ini",
```

Comment:
it should be fine to leave this in for the ppl that use the mypy extension anyways

Reply:
Alr lemme resurrect it quickly.